specs/005-crd-auth-ui-tests/contracts/crd-auth-selector-contract.md

```
@@ -0,0 +1,74 @@
+## Contract: required hooks per CRD screen
+
+### Sign-in screen
+| Element | Required hook (preferred) | Status | Note |
+|---|---|---|---|
+| Email field | textbox, accessible name matching `/e-?mail/i` | _TBD_ | |
+| Password field | textbox/label `Password` | _TBD_ | |
+| Password show/hide toggle | button with accessible name (spec FR-013) | _TBD_ | keyboard-operable |
+| Sign-in submit | button name `Sign in` | _TBD_ | |
+| Forgot-password link | link `/forgot password/i` | _TBD_ | |
+| Sign-up cross-link | link `/sign up/i` or `No account?` text | _TBD_ | |
+| Provider buttons | button per advertised provider, e.g. `/connect with (microsoft|linkedin|github)/i`, in backend order | _TBD_ | order asserted (spec FR-010) |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 4; Actual: 6; Too many cells, extra data will be missing

(MD056, table-column-count)